hybrasyl/Inventory.cs

```diff
@@ -84,8 +85,8 @@ public bool ConditionsValid
             {
                 return _source.Map == _target.Map && _source.IsInViewport(_target) &&
                        _target.IsInViewport(_source) &&
-                       _source.Status.HasFlag(PlayerCondition.InExchange) &&
-                       _target.Status.HasFlag(PlayerCondition.InExchange) &&
+                       _source.Condition.InExchange &&
+                       _target.Condition.InExchange &&

```

Comment:
should this not be an or statement between source and target?

Reply:
No - this is supposed to check that both participants of an exchange are still in it.

Reply:
OK